src/components/picker/nimblePicker.vue

```diff
@@ -38,7 +38,7 @@
     />
     <category
       v-for="category in filteredCategories"
-      v-show="!searchEmojis && (infiniteScroll || category == activeCategory)"
+      v-if="!searchEmojis && (infiniteScroll || category == activeCategory)"
```

Comment:
v-if and v-for should not be mixed.

``
```suggestion
<template v-for="category in filteredCategories">
      <category
        v-if="!searchEmojis && (infiniteScroll || category == activeCategory)"
        ref="categories"
        :key="category.id"
        :data="mutableData"
        :i18n="mutableI18n"
        :id="category.id"
        :name="category.name"
        :emojis="category.emojis"
        :emoji-props="emojiProps"
      />
</template>
```